### DISCUSSION TITLE
Windows: UnicodeDecodeError: 'charmap' codec can't decode byte... when loading bundle.js

### DISCUSSION THREAD

**Discussion by stevej2608** [idom-team/idom#1173] [Problem]
I'm developing a custom JS component. When I test the component it works fine on Linux but when I test on Windows I get *UnicodeDecodeError: 'charmap' codec can't decode byte'..*, It happens when the  bundle is being loaded.

The problem is that fixed by adding utf-8 encoding to loader.

*./reactpy/web/utils.py#L31*
```
    export_names, references = resolve_module_exports_from_source(
        file.read_text(encoding="utf-8"), exclude_default=is_re_export
    )
```

**Comment by Archmonger** [comment]
Would be great if you could PR a solution alongside a regression test.

New issue created:
- #1175
